Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,212 @@
+        let choice = readLine()?.lowercased() ?? "n" // User enter letter choice
+        
+        if choice.count != 1 {
+            print("Please enter one letter at a time.")
+        } else {
+
+//====== maybe add another if statment that checks if the value is a single letter else have them try again? because when i enter 2 letter it says already entered ======//
+            
+            if alphabet.contains(choice) {
+                var spotCounter = 0
+                
+                if word.contains(choice) { // checks if word contains the letter entered
+                    sleep(1); print()
+                    print("Found!")
+                    sleep(1)
+                    animation(wrongCount: wrongCount)
+                    
+                    for char in word { // loops through the word
+                        
+                        if choice == String(char) { // checks if user letter is = to char in word
+                            hiddenWord[spotCounter] = choice
+                        }
+                        spotCounter += 1
+                    }
+                    
+                } else {
+                    print(); sleep(1)
+                    print("Not Found!")
+                    print(); sleep(1)
+                    wrongCount += 1
+                    animation(wrongCount: wrongCount)
+                    print(); sleep(1)
+                    
+                }
+                // Add choice to the already chosen array and print it
+                selectedLetters.append(choice)
+                // print already selected letter joined with " "
+                print("Word: \(hiddenWord.joined(separator:""))       Entered Letters: \(selectedLetters.joined(separator:" "))")
+                
+                // Remove letter from alphabat once chosen:
+                
+                let letterIndex = alphabet.firstIndex(of: choice)
+                alphabet.remove(at: letterIndex ?? 2)
+                // print(alphabet) // Prints alphabets after chosen letters have been removed.
+                
+            } else {
+                print("Has already been chosen, try again.")
+            }
+            
+           // hiddenWordString = hiddenWord.joined(separator:"")
+        }
+        hiddenWordString = hiddenWord.joined(separator:"")
+
+    } while wrongCount < maxWrong && word != hiddenWordString
+
+// END OF GAME OUTPUT: YOU WIN OR LOSE
+
+if wrongCount >= maxWrong {
+    print()
+    print("GAME OVER - YOUVE BEEN HUNG 😵")
+    print("The correct word was: \(word)")
+} else if word == (hiddenWord.joined(separator:"")) {
+    print(); sleep(1)
+    print("YAY! You win 👏🏽 👏🏽 👏🏽, it took \(selectedLetters.count) tries ") //\(guessCount)
+}
+    print()
+ 
+    print("Would you like to play again? Enter yes or no ")
+    playAgain = readLine()?.lowercased() ?? "no"
+    
+ } while playAgain == "yes"
+print(); sleep(1)
+print("Thanks for playing. 👋🏽")
+
```

Comment:
When the game restarts, it shows the previous word that was last played, make sure to reset the game in the future :)